devtools/datasette/fly/staging.toml

```diff
@@ -0,0 +1,27 @@
+# fly.toml app configuration file generated for catalyst-coop-pudl-staging on 2024-06-12T10:41:45-04:00
+#
+# See https://fly.io/docs/reference/configuration/ for information about how to use this file.
+#
+
+app = "catalyst-coop-pudl-staging"
+primary_region = "bos"
+
+[[mounts]]
+  destination = "/data"
+  source = "datasette"
+
+[http_service]
+  internal_port = 8080
+  force_https = true
+  auto_stop_machines = true
+  auto_start_machines = true
+  min_machines_running = 0
+  processes = ["app"]
+
+[[vm]]
+  memory = "1gb"
```

Comment:
I guess 1GB is enough since it's not going to load the whole SQLite DBs into memory, but each of the DBF and XBRL derived Form 1 DBs are close to 1GB (which is kind of wild since one of them covers 27 years, and the other covers 2 years)

Reply:
It's easy enough to change if necessary, but I figured most of the staging stuff is gonna be with small subsets of the data anyways.

Reply:
Slash, it totally worked for my weird little nginx journey today.